CHANGES.md

```diff
@@ -1,5 +1,19 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.6.1
+
+* Rollback of time-related functions dispatching on `Dispatchers.Main`.
+   This functionality was introduced in 1.6.0 and was found inconvenient and erroneous (#3106, #3113).
+* Reworked newly-introduced `CopyableThreadContextElement` to solve uncovered issues after the initial release (#3227).
+* Fixed a bug when `ThreadLocalElement` was not properly updated in racy scenarios (#2930).
```

Comment:
```suggestion
* Fixed a bug with `ThreadLocalElement` not being properly updated in racy scenarios (#2930).
```
Didn't see an established form "a bug when". "With", "where" are common. https://forum.wordreference.com/threads/fixed-the-bug-where-which-that-some-pictures-dont-show-up.3320349/ found a discussion with some other examples.

The same is true for some of the entries below.